onnxruntime/test/debug_node_inputs_outputs/debug_node_inputs_outputs_utils_test.cc

```diff
@@ -48,7 +48,7 @@ void VerifyTensorProtoFileDataInt4(const PathString& tensor_proto_path,
 
   std::vector<Int4x2Base<Signed>> actual_data{};
   actual_data.resize(expected_data.size());
-  ASSERT_STATUS_OK(utils::UnpackTensor(tensor_proto, Path{}, actual_data.data(), num_elems));
+  ASSERT_STATUS_OK(utils::UnpackTensor(tensor_proto, std::filesystem::path(), actual_data.data(), num_elems));
```

Comment:
std::filesysstem::path(tensor_proto_path) ?

Reply:
The path is model path, where the model file lives. Here we do not have a model.